src/store/autoLogout.ts

```diff
@@ -180,8 +181,8 @@ export class AutoLogoutModule extends VuexModule {
 
 	@Mutation
 	setInit(
-		showWarningOnRemainingSeconds: number,
-		defaultRemainingTimeInSeconds: number
+		showWarningOnRemainingSeconds?: number,
+		defaultRemainingTimeInSeconds?: number
 	) {
 		this.showWarningOnRemainingSeconds = showWarningOnRemainingSeconds || 3600;
```

Comment:
this does not seem to be solid: the default value is set already in https://github.com/hpi-schul-cloud/nuxt-client/blob/e34cc71b6c30b5617d53475a973bdae033fd262b/src/store/autoLogout.ts#L150 on initialization. therefore the value must be given when creating a new instance of a new AutoLogoutModule() immediately. 
when needed, the method should call a prop setter which ensures the no invalid values can be given.

Reply:
```suggestion
if(showWarningOnRemainingSeconds !== undefined){
		this.showWarningOnRemainingSeconds = showWarningOnRemainingSeconds;
}
```
on this.showWarningOnRemainingSeconds a setter could be added with property validation like  is number and gt zero